core/templates/app.constants.ts

```diff
@@ -102,6 +103,8 @@ export const AppConstants = {
     UNICODE_STRING: 'UnicodeString',
   },
 
+  DEST_IF_REALLY_STUCK_FEAT_ENABLED: true,
```

Comment:
This should be false for now. Also, add a comment mentioning exactly what this flag controls.

Reply:
Done.